get-resource.sh

```diff
@@ -15,21 +15,12 @@ FFILENAME=$FILENAME$FILENAME_EXT
 mkdir -p /shared/html/images /shared/tmp
 cd /shared/html/images
 
-TMPDIR=$(mktemp -d -p /shared/tmp)
-
 # Is this a RHEL based image? If so the IPA image is already here, so
 # we don't need to download it
-if [ -e /usr/share/rhosp-director-images/ironic-python-agent-latest.tar ] ; then
-    VERSION=$(cat /usr/share/rhosp-director-images/version-latest.txt)
```

Comment:
I wonder should we keep the version somewhere? There is now nothing in the container image to say what version of the IPA image is/was.

Reply:
considering that we're using a modified version of that image, I'm not sure only this could give enough info
I'll see to add a way to track all the versions of the packages modified